### DISCUSSION TITLE
posts/compile-program-in-vim

### DISCUSSION THREAD

**Discussion by giscus** [EarthMessenger/blog.earthmessenger.xyz#12] [giscus]
# 2023/02/26/compile-program-in-vim

新手：直接使用 map 映射编译命令

https://blog.earthmessenger.xyz/2023/02/26/compile-program-in-vim.html

**Comment by fat-old-eight** [comment]
%%%

**Comment by fat-old-eight** [comment]
不如Error Lens+手打命令行
